lib/libzrepl/data_conn.c

```diff
@@ -718,10 +733,11 @@ uzfs_zvol_rebuild_dw_replica(void *arg)
 			 */
 			zvol_rebuild_status_t state;
 			state = uzfs_zvol_get_rebuild_status(zinfo->main_zv);
-			if (state != ZVOL_REBUILDING_AFS) {
+			if (state != ZVOL_REBUILDING_AFS &&
```

Comment:
these conditions should be fine. How about checking just for ZVOL_REBUILDING_SNAP? Give a thought.. I hadn't thought of it completely